src/views/workflow-history/workflow-history-event-status-badge/__tests__/workflow-history-tab-event-status-badge.test.tsx.snapshot

```diff
@@ -3,7 +3,7 @@
 exports[`WorkflowHistoryEventStatusBadge should match snapshot when size is not valid and icon should not be rendered 1`] = `
```

Comment:
Going to look for the fix mentioned here
https://styletron.org/react#snapshot-testing